src/vote/vote.mongo.service.ts

```diff
@@ -172,23 +172,23 @@ export class VoteMongoService {
         if (process.env.NODE_ENV === 'development') this.logger.debug(`votePowers aggregate: \n${JSON.stringify(votePowers)}`);
 
         const sumVotePowers = votePowers.map((poll_option) => {
-            let sum = ethers.BigNumber.from('0');
+            let sum = 0n;
```

Comment:
is this a common way to describe 0? Haven't seen it before

Reply:
It is called literals for denoting BigInt ecma native data type. Ethers switched from their own implementation to handle big numbers to ecma2020